go_api/usecase/admin_usecase.go

```diff
@@ -0,0 +1,64 @@
+// AdminのUsecaseを定義する
+package usecase
+
+import (
+	"github.com/NUTFes/lottery/go_api/domain"
+)
+
+type adminUsecase struct {
+	adminRepository domain.AdminRepository
+}
+
+type AdminUsecase interface {
+	FindAllAdmin() (*domain.Admins, error)
+	FindAdmin(id int) (*domain.Admin, error)
+	CreateAdmin(admin *domain.Admin) error
+	UpdateAdmin(admin *domain.Admin) error
+	DeleteAdmin(id int) error
+}
+
+func NewAdminUsecase(ur domain.AdminRepository) AdminUsecase {
+	return &adminUsecase{adminRepository: ur}
+}
+
+// 全アドミンユーザーの取得
+func (u *adminUsecase) FindAllAdmin() (*domain.Admins, error) {
+	admins, err := u.adminRepository.FindAll()
+	if err != nil {
+		return nil, err
+	}
+	return admins, nil
+}
```

Comment:
```suggestion
func (u *adminUsecase) FindAllAdmin() (*domain.Admins, error) {
	admins, err := u.adminRepository.FindAll()
	if err != nil {
		return nil, err
	}
	return admins, nil
}
```
このファイルもレシーバーの修正をお願いします